code_samples/api/rest_api/config/routes.yaml

```diff
@@ -0,0 +1,9 @@
+app.rest:
+    resource: routes_rest.yaml
+    prefix: '%ibexa.rest.path_prefix%'
+
+app.rest.options:
+    #TODO: There is something wrong with resource, it only works with an absolute path on ibexa/rest v4.1.2 — see https://issues.ibexa.co/browse/IBX-2927
```

Comment:
@DominikaK TODO left with a question:
Do I remove/comment the "OPTIONS method support" section until this is fixed, or do I let it as it should be?


In hindsight, I would now say the second, I just remove the TODO and let the doc says how it should work event if there is a actually a reported bug.

Reply:
I would say: depending on how likely the bug is to be fixed quickly. If it isn't, it may be better to remove the mention in doc until the issue is resolved.

Reply:
I removed it in 81992da6abe9a67f6b8ef68a8b58773a5da5361e and will propose it back into a new PR linked to [IBX-2927](https://issues.ibexa.co/browse/IBX-2927)